sql/2019/10_SEO/10_03.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+
+# <link rel="amphtml"> (AMP)
+
+CREATE TEMP FUNCTION hasAmpLink(payload STRING)
+RETURNS BOOLEAN LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(payload);
+  var almanac = JSON.parse($._almanac);
+  return !!almanac['link-nodes'].find(node => node.rel.toLowerCase() == 'amphtml')
+} catch (e) {
+  return false;
+}
+''';
+
+SELECT
+  COUNTIF(hasAmpLink(payload)) AS score_sum,
+  COUNTIF(hasAmpLink(payload)) / COUNT(0) AS score_percentage
```

Comment:
I think we want all the percentage queries as `ROUND(numerator * 100 / denominator, 2)`